<dec f='llvm/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h' l='665' type='const char * lldb_private::python::PythonException::toCString() const'/>
<def f='llvm/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp' l='1015' ll='1019' type='const char * lldb_private::python::PythonException::toCString() const'/>
<use f='llvm/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp' l='1041' u='c' c='_ZN12lldb_private6python15PythonExceptionC1EPKc'/>
<use f='llvm/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp' l='1043' u='c' c='_ZN12lldb_private6python15PythonExceptionC1EPKc'/>
<use f='llvm/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp' l='1049' u='c' c='_ZN12lldb_private6python15PythonException7RestoreEv'/>
<use f='llvm/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp' l='1061' u='c' c='_ZNK12lldb_private6python15PythonException3logERN4llvm11raw_ostreamE'/>
<use f='llvm/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp' l='1087' u='c' c='_ZNK12lldb_private6python15PythonException13ReadBacktraceB5cxx11Ev'/>
<use f='llvm/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp' l='1097' u='c' c='_ZNK12lldb_private6python15PythonException13ReadBacktraceB5cxx11Ev'/>
